api/chain.go

```diff
@@ -541,10 +595,14 @@ func (a *API) chainTxByIndexHandler(msg *apirest.APIdata, ctx *httprouter.HTTPCo
 
 // chainTxByHeightHandler
 //
-//	@Summary		Returns the list of transactions for a given block
+//	@Summary		Block transactions
```

Comment:
may i suggest `Transactions in a block`?

it's just a suggestion because maybe `Block transactions` does fit in better in the actual context (i don't know), but without context it sounds to me like blocking a transaction.

Reply:
0ad7a1a5